graphql_client_codegen/src/codegen_options.rs

```diff
@@ -68,6 +70,7 @@ impl GraphQLClientCodegenOptions {
             extern_enums: Default::default(),
             fragments_other_variant: Default::default(),
             skip_serializing_none: Default::default(),
+            serde_path: syn::parse_quote!(::serde),
```

Comment:
Given that the generated code is already dependent on `::graphql_client`, I don't think this has to be configurable. Do you have a usecase for having this configurable?

Reply:
The `graphql_client_codegen` crate is used outside of the derive macro (see its [reverse dependencies](https://crates.io/crates/graphql_client_codegen/reverse_dependencies)). Unilaterally changing the path used for serde imports would break users who are generating code for an older `graphql_client` version.

It's been a while since I wrote the code here but I think that was the reason that I did it as an option here.

Reply:
> The `graphql_client_codegen` crate is used outside of the derive macro (see its [reverse dependencies](https://crates.io/crates/graphql_client_codegen/reverse_dependencies)). Unilaterally changing the path used for serde imports would break users who are generating code for an older `graphql_client` version. Notably, it would change the output of the CLI.

You are right, this causes the least breaking change. The problem I see is that a config option is not good for discoverability. However, if you are using the codegen crate, then you are probably an advanced user. I think it makes sense to do it this way.